test/multiverse/lib/multiverse/suite.rb

```diff
@@ -67,6 +67,10 @@ def filter_file
       opts.fetch(:file, nil)
     end
 
+    def infinite_tracing_suite?
+      suite == 'infinite_tracing'
+    end
```

Comment:
For this file, I wanted to be able to able to perform:

```shell
bundle exec rake test:multiverse[infinite_tracing,file=config_test,debug] 
```

but it was incorrectly scoping my `config_test` file filter to the `test/multiverse/suites/infinite_tracing` directory. Now using `file=` will work as desired for 8T tests.